.github/workflows/release.yml

```diff
@@ -0,0 +1,14 @@
+name: Release
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: marvinpinto/action-automatic-releases@v1.2.1
+        with:
+          repo_token: "${{ secrets.GITHUB_TOKEN }}"
+          automatic_release_tag: "latest"
```

Comment:
can we change this bit to use the same name as the tag?

Reply:
Sorry as the PR was closed made a new one with the requested change